blocked-edges/4.12.0-rc.0-PodmanMissingAuthFlag.yaml

```diff
@@ -0,0 +1,21 @@
+to: 4.12.0-rc.0
+from: 4[.]11[.].*
+url: https://issues.redhat.com/browse/MCO-540
+name: PodmanMissingAuthFlag
+message: |-
+  4.12 started using -authfile flag with podman to perform in-place upgrade on nodes and it does not work with nodes installed with OCP 4.1 version. This risk does not apply if a cluster was installed with version 4.2 or later.
```

Comment:
nit: [`--authfile`][1] with two dashes?

[1]: https://github.com/containers/podman/blob/4abc164f5613b42c2060fffed70cecba1beb2fee/docs/source/markdown/options/authfile.md#--authfilepath

Reply:
Fixed it.